CHANGELOG

```diff
@@ -1,5 +1,7 @@
 Changes in Version 0.2.3 (2021-xx-xx)
 =====================================
+toolbox
+ - docstrings cleaned for numpydoc format and Raises documented where it added value
```

Comment:
Over 80 character, wrapping would be nice (it's inconsistent in CHANGELOG, alas.)

Reply:
@jtniehof There was a conversation here a bit ago. Are we doing 80? Seems really small, I can't remember the outcome.